odoo/models.py

```diff
@@ -45,6 +45,7 @@
 import psycopg2, psycopg2.extensions
 from lxml import etree
 from lxml.builder import E
+from werkzeug import urls
```

Comment:
(this will be removed following my other comment but so you know, keep everything sorted alphabetically)